.travis.yml

```diff
@@ -0,0 +1,15 @@
+os: osx
+language: objective-c
+osx_image: xcode8.3
+node_js: 5.1.1
+
+before_install:
+  - cd server
+
+install:
+  - npm install
+
+script:
```

Comment:
Yes indeed! 

I broke it apart because I don't have a web browser running on Travis yet. I'll update it to just `npm test`. I'll figure out browsers-on-travis later today, I hope.

(You can see my attempts on my fork, [hawkrives/uebersicht#travis](https://github.com/hawkrives/uebersicht/tree/travis), and on [TravisCI itself](https://travis-ci.org/hawkrives/uebersicht).)